odoo_initializer/models/drug_loader.py

```diff
@@ -1,15 +1,15 @@
 import logging
 
-from .base_loader import BaseModelImporter
+from .base_loader import BaseModelLoader
 
 _logger = logging.getLogger(__name__)
 
 
-class DrugLoader(BaseModelImporter):
+class DrugLoader(BaseModelLoader):
     update_existing_record = True
     data_files_source = "openmrs"
     model_name = "product.template"
-    field_mapping = {"name": "Fully specified name:en", "lst_price": "odoo_price"}
-    folder = "concepts"
+    field_mapping = {"name": "name", "drug_name": "Fully specified name:en", "lst_price": "odoo_price"}
```

Comment:
@chinzou , `"drug_name": "Fully specified name:en"`: is there a field named `drug_name` in the Odoo model?

Reply:
no, it's a field that is supposed to be added by bahmni based on the previous version. should we wait until the new version become available for test and we base on that ?